app/src/main/res/layout/activity_map.xml

```diff
@@ -8,25 +8,23 @@
         android:layout_height="match_parent"
         tools:context=".MapActivity">
 
-        <Button
-            android:id="@+id/button_get_location"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="@string/get_location"
+        <TextView
+            android:id="@+id/location_display"
+            android:layout_width="307dp"
+            android:layout_height="91dp"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintTop_toBottomOf="@+id/mapView" />
 
-        <TextView
-            android:id="@+id/location_display"
-            android:layout_width="183dp"
-            android:layout_height="158dp"
-            app:layout_constraintBottom_toBottomOf="parent"
+        <com.google.android.gms.maps.MapView
+            android:id="@+id/mapView"
+            android:layout_width="407dp"
+            android:layout_height="391dp"
```

Comment:
It would be better not to hard-code the dimensions, maybe you can use match_parent for the width.
For the height, I'm not sure how. Maybe the height of the TextView could be wrap_content and the map use all the space left.

Reply:
You're right I will fix that, thanks!